community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/CostBasedExecutablePlanBuilder.scala

```diff
@@ -49,7 +49,7 @@ case class CostBasedExecutablePlanBuilder(monitors: Monitors,
                                 plannerName: CostBasedPlannerName,
                                 runtimeName: RuntimeName,
                                 useErrorsOverWarnings: Boolean,
-                                showWarnings: Boolean)
+                                userHasSpecifiedRuntime: Boolean)
```

Comment:
Since you have the `runtimeName` passing in the boolean `userHasSpecifiedRuntime` looks pointless, can we check in here if `runtimeName == FallbackRuntime`?